pkg/cmdutil/cmd.go

```diff
@@ -0,0 +1,80 @@
+package util
```

Comment:
looks like this whole file is only used in oc

Reply:
There are a lot of stuff here, and majority is somewhat oc related, I'd not move it, just yet.